src/components/Video/_video.scss

```diff
@@ -0,0 +1,22 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-video {
+  aspect-ratio: var(--aspect-ratio);
+  display: block;
+  height: auto;
+  position: relative;
+  width: 100%;
+
+  &__container {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
I think lines 10-13 are unnecessary since the iframe is absolutely positioned? I would probably just explicitly set the `left` and `top` of the iframe.